mitodl/mitxpro
# Remove tox, move python test and linting to ./travis/python_tests.sh

#### What are the relevant tickets?
None

#### What's this PR do?
Removes tox and moves the python linting and pytest into a script in the `travis` directory. 

I did not touch pylint or pep8 but I'm happy to include moving away from the pytest plugins if people think it fits in the PR.

#### How should this be manually tested?
The travis output should look like the tests and linting is being run. Travis tests should pass. You should be able to run `docker-compose run web pytest` and have the tests run as expected.
